Makefile

```diff
@@ -26,6 +28,18 @@ else
 	composer update --prefer-dist
 endif
 
+handlebars:
+ifeq (,$(handlebars))
+	@echo "No handlebars command available, remember to install this"
+	@echo " # npm install handlebars -g"
+	@echo "Then you must compile the templates"
+	@echo " # make js-templates"
+else
+	@echo "Remember compile the templates"
```

Comment:
why we need to tell user to do this? Any reason why `js-templates` cannot be just another rule, something like this:
```
javascript_deps: js/handlebars.js js/lozad.js js/templates.js

js/templates.js:
	handlebars js/templates -f js/templates.js
```

Reply:
The dependency is the handlebars binary. :sweat_smile: 
Let it be clear that this is only in development. Users from the appstore, nor should they know about this.